src/chatshare.py

```diff
@@ -28,6 +32,16 @@ def main():
     """
     print("Welcome to Chatshare!")
 
+    # Get all servers
```

Comment:
I think we need to be smarter here - specifically if a new server starts, or one stops, then we need some kind of mechanism to handle that. Perhaps we need a 'MinecraftServerManager' class which handles the starting and stopping of servers based on checks against Pelican regularly (every 5 mins?).